identity-provider/src/pages/Dashboard.jsx

```diff
@@ -0,0 +1,171 @@
+
+// List of organizations and database/api calls:
+// 1. Setup list of user's shared info to organizations
+// 2. Once that is set up it can be implemented with the database and api is set up for that 
+
+// Requests:
+// 3. We only need one get request for the dashboard organizations (that show up) page to get all of the linked organizations
+// 4. For the 'unlink' button we will need to do a patch request to update that list of organizations for that user once the confirm button is pressed.
+
+// Dialog:
+// 5. The confirm can be the same dialog section as we did for the DashboardOrganization function and will send that patch request
+// 6. Piggybacking from #5: We need to create the alert box that shows the users information when the 'edit info' button is pressed as well as the cancel and save buttons
+// 7. Piggybacking from #6, we need to create the confirmation unlink box once you press the 'unlink' button, as well as the cancel and confirm buttons  
+
+
+// Questions:
+// 1. How do we "update" a page in the background, such as the website detecting whether the user is connected to organizations or not: GET requests?
+// 2. With that said, how does the software know what to display every time the user logs in? GET requests?
+// 5. The "edit info" and "unlink" buttons as well as the organization info will have to dissapear from the dashboard when the user presses 'unlink'.
+// Pop up alerts:
+// The "Your Information" alert will contained stored data from the settings page(whatever info is updated from pressing the "update profile" button.)
+// How to we show the dashbooard no organizations and the dashboard organizations page? Depends on what the if else statements say and what button the user presses?
+
+
+// Imports from material ui page, should they be within the DashboardOrganizations function or here at the top?
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+// ******************************************************
+
+function DashboardNoOrganizations() {
+
+    const voulenteerButton = () => {
+        console.log('This button will link to our relying parties website')
+    }
+    return (
+        <>
+        <h1>You're not currently connected to any organizations.</h1>
+        <p>Search for voulenteer oppertunities <button type = "submit" onClick = {voulenteerButton}> Here</button></p>
+        
+        </>
+    )
+}
+
+function DashboardOrganizations() {
+
+   // code copied from material ui: Will show the dialog component on the website
+  //  Will need a seperate one for each dialog's title and description
+  //  Will need another open, setOpen for the other dialog
+
+
+   const [open, setOpen] = React.useState(false);
```

Comment:
This state variable would be renamed to something like openViewInfo and setOpenViewInfo